src/sign.c

```diff
@@ -2165,151 +2128,154 @@ set_context_in_sign_cmd(expand_T *xp, char_u *arg)
 
     end_subcmd = skiptowhite(arg);
     if (*end_subcmd == NUL)
-	// expand subcmd name
-	// :sign {subcmd}<CTRL-D>
-	return;
+        // expand subcmd name
+        // :sign {subcmd}<CTRL-D>
+        return;
 
     cmd_idx = sign_cmd_idx(arg, end_subcmd);
 
     // :sign {subcmd} {subcmd_args}
-    //		      |
-    //		      begin_subcmd_args
+    //                |
+    //                begin_subcmd_args
     begin_subcmd_args = skipwhite(end_subcmd);
 
     // expand last argument of subcmd
 
     // :sign define {name} {args}...
-    //		    |
-    //		    p
+    //              |
+    //              p
 
     // Loop until reaching last argument.
     p = begin_subcmd_args;
     do
     {
-	p = skipwhite(p);
-	last = p;
-	p = skiptowhite(p);
-    } while (*p != NUL);
+        p = skipwhite(p);
+        last = p;
+        p = skiptowhite(p);
+    }
+    while (*p != NUL);
```

Comment:
That while on a separate line looks  also a bit unusual.

Reply:
Done